### PR TITLE
Newdepclient

### PR DESCRIPTION
**Pull request by jkagliwal-px**
https://jenkins.pwx.dev.purestorage.com/job/Users/job/Madan/job/platfrom-build-torpedo/599/console

testrun - https://jenkins.pwx.dev.purestorage.com/job/PDS/job/pds2.0-byoc-branch-build-test/521/console

### CODE REVIEW

**Review by aravindranath14 — APPROVED** [review]
LGTM

**Review by Madanagopal19 — CHANGES_REQUESTED** [review]
Run Make and Merge the PR

**Comment by jkagliwal-px** [comment]
> Run Make and Merge the PR

Done.

File: drivers/unifiedPlatform/pds/backend/v1/api/restore.go [hunk 1]
```diff
@@ -115,22 +115,22 @@ func (restore *PDS_API_V1) GetRestore(getRestoreRequest *automationModels.PDSRes
...
-	deleteRequest := restoreClient.RestoreServiceDeleteRestore(ctx, deleteRestoreRequest.Delete.Id)
-
-	_, res, err := deleteRequest.Execute()
-	if err != nil || res.StatusCode != status.StatusOK {
-		return fmt.Errorf("Error when calling `RestoreServiceGetRestore`: %v\n.Full HTTP response: %v", err, res)
-	}
-
-	return nil
-}
+//// DeleteRestore will delete restore for a given deployment
```
Comment (Madanagopal19) [inline drivers/unifiedPlatform/pds/backend/v1/api/restore.go:118]: these commented codes are not required ?

**Review by Madanagopal19 — CHANGES_REQUESTED** [review]

File: drivers/unifiedPlatform/pds/backend/v1/api/restore.go (hunk 1 above)
Comment (jkagliwal-px) [reply drivers/unifiedPlatform/pds/backend/v1/api/restore.go:118]: No, Delete Restore is removed. But I have commented for now, if its needed later.

**Review by atrivedi-px — APPROVED** [review]
LGTM

**Pull request merged**